vscode-antimony/src/server/main.py

```diff
@@ -141,6 +145,30 @@ def sbml_file_to_ant_file(ls: LanguageServer, args):
             'file': full_path_name
         }
 
+@server.thread()
+@server.command('antimony.sendType')
+def get_type(ls: LanguageServer, args):

```

Comment:
This function returns a value and so it MUST have a return type. Of course, you need a description comment as well.

Reply:
@longxf21 can you fix this?